backend/src/main/java/gov/cdc/usds/simplereport/db/model/Organization.java

```diff
@@ -33,6 +34,19 @@ public Organization(String orgName, String orgType, String externalId, boolean i
     this.identityVerified = identityVerified;
   }
 
+  @Override
+  public boolean equals(Object o) {
```

Comment:
adding this here to get the service tests (which need to check for org equality) to pass correctly.